sdk/jdbc/tests.yml

```diff
@@ -3,11 +3,15 @@ trigger: none
 stages:
     - template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
       parameters:
-          SupportedClouds: 'Public,UsGov,China'
+          SupportedClouds: 'Public'
+          Clouds: 'Public'
           TestResourceDirectories:
           Artifacts:
+              - name: azure-identity-providers-jdbc-mysql
+                groupId: com.azure
+                safeName: azureidentityprovidersjdbcmysql
```

Comment:
Why this?

Reply:
We have pipeline `java - jdbc - tests` and `java - jdbc - tests - weekly`, and always fail, because we have no IT for jdbc, but it seems I can't delete these two pipelines, to pass these two pipelines I add this.

Reply:
Then should we delete this tests pipeline?

Reply:
I have tried to delete these tests pipeline, but it will recover automatically.

Reply:
Even after removing the tests.yaml?